packages/manager/modules/up-sell-cross-sell/src/advices/template.html

```diff
@@ -0,0 +1,25 @@
+<oui-tile
+    heading="{{ ::'uscs_heading' | translate }}"
+    description="{{ ::$ctrl.description }}"
+    data-ng-if="$ctrl.advicesAvailable"
+>
+    <div data-ng-repeat="advice in $ctrl.advices track by $index">
+        <oui-tile-button
+            data-ng-if="advice.href"
+            href="{{ ::advice.href }}"
```

Comment:
**suggestion**:
Maybe using `ng-attr-href` could help you to not duplicate this `oui-tile-button`?